library/x509_crl.c

```diff
@@ -546,15 +546,10 @@ int mbedtls_x509_crl_parse( mbedtls_x509_crl *chain, const unsigned char *buf, s
     {
```

Comment:
The docs in the declarations of `mbedtls_x509_crl_parse_der` and `mbedtls_x509_crl_parse` in `include/mbedtls/x509_crl.h` need updating - `buflen` no longer needs to allow for a terminating null byte.

Same comment applies to the crt and csr functions.